### PR TITLE
Connect the palettes to the project names based off of the primary key.

### PR DESCRIPTION
**Pull request by Jessiewithani**
I also regenerated the palette different to where it shows the color tiles as well.

#### What's this PR do?
changed the format around as to where the projects are being generated. Also connected the palettes to the correct project name.
#### Where should the reviewer start?
you can view the changes made in ProjectContainer, Project, and Palette.
#### Questions:
- Is there a blog post?
- Does the knowledge base need an update?
- Does this add new dependencies which need to be added?

### CODE REVIEW

File: src/Project/Project.js [hunk 1]
```diff
@@ -4,33 +4,48 @@ import './Project.scss';
...
+      
+
+    } 
+      
     return(
       <>
-        <h3 className="project_name">{displayNames}</h3>
-          {displayPalettes}
+        <h3 className="project_name">{name}}</h3>
+          {/* {displayPalettes} */}
```
Comment (allisonjw) [inline src/Project/Project.js:40]: On line 39 there is an extra '}' after name. Also, this should be passed in as palette_name to show up.

File: src/Palette/Palette.js [hunk 2]
```diff
@@ -2,30 +2,45 @@ import React from 'react';
 import './Palette.scss';
 import trashBin from '../images/rubbish.png'
 
-const Palette = ({ palette_name, color_1, color_2, color_3, color_4, color_5 }) => {
-  // add removePalette or pass down from ProjectContainer?
-  // if created here import apicall, usedispatch and action
-    const colors =[color_1, color_2, color_3, color_4, color_5];
+const Palette = ({name, id, color_1, color_2, color_3, color_4, color_5}) => {
```
Comment (allisonjw) [inline src/Palette/Palette.js:6]: On line 5, it should actually be palette_name

**Review by allisonjw — CHANGES_REQUESTED** [review]
@Jessiewithani 
The name being passed down to palettes needs to match the data to be displayed. So {palette_name}, not {name}.If you could change that then I can merge it. Thank you!

File: src/Project/Project.js (hunk 1 above)
Comment (Jessiewithani) [reply src/Project/Project.js:40]: good lookout, I changed it.

File: src/Palette/Palette.js (hunk 2 above)
Comment (Jessiewithani) [reply src/Palette/Palette.js:6]: i changed this as well

File: src/Project/Project.js (hunk 1 above)
Comment (Jessiewithani) [reply src/Project/Project.js:40]: yup saw this. Thank you.

**Pull request merged**